packages/taquito-beacon-wallet/README.md

```diff
@@ -26,23 +26,27 @@ import { BeaconWallet } from '@taquito/beacon-wallet';
 const options = {
   name: 'MyAwesomeDapp',
   iconUrl: 'https://taquito.io/img/favicon.svg',
-  preferredNetwork: 'chosen_network',
+  network: { type: 'ghostnet' },
   eventHandlers: {
     PERMISSION_REQUEST_SUCCESS: {
       handler: async (data) => {
         console.log('permission data:', data);
       },
     },
   },
+  enableMetrics: true,
 };
 const wallet = new BeaconWallet(options);
 
-// The Beacon wallet requires an extra step to set up the network to connect to and the permissions:
-await wallet.requestPermissions({
-  network: {
-    type: 'chosen_network',
+await wallet.requestPermissions();
+
+wallet.client.subscribeToEvent(
```

Comment:
looks like `subscribeToEvent` is returning a `Promise<void>`,  would be safe to add `await` in front

Reply:
they also don't have `await` in front of `requestPermission() either`.  And i think dealing with promises in sequence is better to `await` each other out

Reply:
On that note, I think it's supposed to `subscribeToEvent` first then `requestPermission` like their example